e2e/tests/deploy/testdata/tanka/devspace.yaml

```diff
@@ -0,0 +1,17 @@
+version: v2beta1
+name: tanka-deploy
+vars:
+  IMAGE_NGINX: nginx:alpine
+  TANKA_PATH:
+    command: pwd
+
+deployments:
+  # Define a tanka deployment
+  nginx-test:
+    tanka:
+      runJsonnetBundlerInstall: true
+      runJsonnetBundlerUpdate: true
+      path: ${TANKA_PATH}
```

Comment:
Why is that needed, can we not make this the default?

Reply:
We can make this as default regarding `runJsonnetBundlerInstall` and `runJsonnetBundlerUpdate` 
With more experience, I think we should remove it from the code and pute all jsonnet bundler stuff into a devspace function, it mainly depends on how you see it @FabianKramm

Reply:
+1 on this -- the answer is: it depends; the `install` is a bootstrap that ensures dependencies are installed.

the `update` retrieves the latest versions of linked dependencies. This might be useful for us internally, as we want to pull the latest (stable) versions of e.g. an external dependency in, but it is definitely not a general rule of thumb.

Reply:
So it depends on what should be the recommended flow, if you think your flow is an exception here, we can have it default off, but it seems to me at least install would be required as this has otherwise executed manually

Reply:
I've added the `install = true` as default, the update operation is more specific and custom to our workflow and therefore `false` by default now